data/remnant/remnant 2 cognizance.txt

```diff
@@ -657,7 +657,7 @@ mission "Remnant: Cognizance 18"
 		event "remnant: pantica void sprites"
 		event "remnant: cognizance timer" 30
 		conversation
-			`Moments later a ping comes in on the commlink, and Prefect Chilia and Plume appear onscreen. "Thank you for your work, Captain. It is unclear what the results of this effort will be, but for now, we wait and see. Hopefully the void sprites settle in," concludes Plume.`
+			`Moments later a ping comes in on the commlink, and Prefect Chilia and Plume appear on screen. "Thank you for your work, Captain. It is unclear what the results of this effort will be, but for now, we wait and see. Hopefully the void sprites settle in," concludes Plume.`
```

Comment:
This looks and sounds wrong to me.

According to https://www.merriam-webster.com/dictionary/on-screen it should be either `onscreen` or `on-screen`, with https://dictionary.cambridge.org/dictionary/english/on-screen suggesting that it could be `on the screen`. I do not see anywhere suggesting that `on screen` is proper grammar.r

Reply:
I had heard from tibetiroka in a different PR that this was incorrect as it was.
I'll go with on-screen, because I think that's reasonable to both sides?

Reply:
I didn't verify that it was incorrect.

Reply:
Prior to the changes in this PR:
"onscreen" appeared once in game content and once in a code comment.
"on-screen" appeared twice in code comments.
"on screen" appeared three times in game content, and eleven times in code comments.

Reply:
So are you saying I should or shouldn't change it to on-screen?

Reply:
Out of a collection of different but correct options, the most important thing is generally to maintain consistency.
So far, the only case for "on screen" not being correct that has been made is a lack of statement that it is correct. Unless there is a better argument, I don't think this is enough to move away from the de facto standard we've had for almost a decade.

Reply:
The goal of typo fixes is to improve the text, not just maintain the status quo. Based on the reading in the various dictionaries, I would say that basically anywhere that has "on screen" should be replaced by one of the following:
- onscreen
- on-screen
- on the screen

"on screen" itself is uncommon and I suppose technically OK, but doesn't have a good flow to it, and is generally not the accepted way to say this. I think there's a valid case to be made for picking one of "onscreen" or "on-screen" as those are basically the same thing; with "on the screen" being a different way of phrasing it that is more specific, but that gets into word choice and not spelling choice.